README.md

```diff
@@ -1,11 +1,14 @@
 # ActiveRecord Cloud Spanner Adapter
 
-# 🚨THIS CODE IS STILL UNDER DEVELOPMENT🚨
-
 [Google Cloud Spanner](https://cloud.google.com/spanner) adapter for ActiveRecord.
 
 ![rubocop](https://github.com/googleapis/ruby-spanner-activerecord/workflows/rubocop/badge.svg)
 
+This project provides a Cloud Spanner adapter for ActiveRecord. It has the __Preview__ release status and supports ActiveRecord 6.0.x/6.1.x with Ruby 2.6 or higher.
```

Comment:
It might also be good to explicitly mention here that ActiveRecord 6.0.x is not compatible with Ruby 3.0.

The supported versions are:
- ActiveRecord 6.0.x with Ruby 2.5, 2.6 and 2.7
- ActiveRecord 6.1.x with Ruby 2.5 and higher.

(But as mentioned above, I'm fine with not communicating the Ruby 2.5 support, as Ruby 2.5. is eol)

Reply:
Updated as suggested. 2.5 is not included though.